src/mlpack/bindings/tests/delete_allocated_memory.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file delete_allocated_memory.hpp
```

Comment:
Is there any difference between this file and the file in ``src/mlpack/bindings/cli/delete_allocated_memory.hpp``? Maybe I missed something.

Reply:
In this case yes, the CLI bindings have to delete the first element of a `tuple<T*, std::string>`, but the test bindings only hold the `T*` directly so the delete command is a little different.

Reply:
I see, thanks for the clarification.